src/test_l1_dsDisplay.c

```diff
@@ -100,23 +98,23 @@ static bool extendedEnumsSupported=false;
 
```

Comment:
dsEEDID_MAX_MON_NAME_LENGTH is not used so remove it

Reply:
done